doc/modules/classes.rst

```diff
@@ -257,7 +257,6 @@ Loaders
    datasets.fetch_kddcup99
    datasets.fetch_lfw_pairs
    datasets.fetch_lfw_people
-   datasets.fetch_mldata
```

Comment:
Same here, maybe still keep the doc even if it's deprecated? Who knows maybe the website will work again in september, and there is still code using this. Those users will no longer be able to find any documentation about this function in the stable docs...   A doc with a large deprecation warning would avoid us some opened issues from confused users.